app/assets/javascripts/components/sample_transfer.js.jsx

```diff
@@ -96,14 +108,35 @@ var SampleTransferModal = React.createClass({
     )
   },
 
+  qcInfoMessage: function(missingQuantity, selectedSamples) {
+    let infoMessage = "There is no Quality Control (QC) info available for these samples"
+    if(missingQuantity > 0 && missingQuantity != selectedSamples.length) {
+      infoMessage = `There is no Quality Control (QC) info available for ${missingQuantity} ${missingQuantity === 1? 'sample' : 'samples'}`
+    }
```

Comment:
Just a very minor readability improvement. I usually don't like multiline ternary operator, but here it would read well:

```js

    const infoMessage = (missingQuantity > 0 && missingQuantity != selectedSamples.length)
      ? `There is no Quality Control (QC) info available for ${missingQuantity} ${missingQuantity === 1 ? 'sample' : 'samples'}`
      : "There is no Quality Control (QC) info available for these samples"
```

Reply:
An `if` with `else` branch would also do... But either is better than a conditional override of the previously assigned value.

Reply:
Yes, in a language where everything is an expression... but this is JS, so I resolve myself to see a multiline ternary!